src/ru/seimanga/src/eu/kanade/tachiyomi/extension/ru/seimanga/SeiManga.kt

```diff
@@ -0,0 +1,197 @@
+        Genre("NC-17(18+)", "el_3969"),
+        Genre("R18+(18+)", "el_3990"),
+    )
+
+    private fun getCategoryList() = listOf(
+        Genre("OEL-манга", "el_6637"),
+        Genre("Додзинси", "el_1332"),
+        Genre("Арт", "el_2220"),
+        Genre("Ёнкома", "el_2741"),
+        Genre("Комикс", "el_1903"),
+        Genre("Манга", "el_6421"),
+        Genre("Манхва", "el_1873"),
+        Genre("Маньхуа", "el_1875"),
+        Genre("Ранобэ", "el_5688"),
+    )
+
+    private fun getGenreList() = listOf(
+        Genre("боевик", "el_1346"),
+        Genre("боевые искусства", "el_1334"),
+        Genre("гарем", "el_1333"),
+        Genre("гендерная интрига", "el_1347"),
+        Genre("героическое фэнтези", "el_1337"),
+        Genre("детектив", "el_1343"),
+        Genre("дзёсэй", "el_1349"),
+        Genre("драма", "el_1310"),
+        Genre("игра", "el_5229"),
+        Genre("исэкай", "el_6420"),
+        Genre("история", "el_1311"),
+        Genre("киберпанк", "el_1351"),
+        Genre("комедия", "el_1328"),
+        Genre("меха", "el_1318"),
+        Genre("научная фантастика", "el_1325"),
+        Genre("омегаверс", "el_5676"),
+        Genre("повседневность", "el_1327"),
+        Genre("постапокалиптика", "el_1342"),
+        Genre("приключения", "el_1322"),
+        Genre("психология", "el_1335"),
+        Genre("романтика", "el_1313"),
+        Genre("самурайский боевик", "el_1316"),
+        Genre("сверхъестественное", "el_1350"),
+        Genre("сёдзё", "el_1314"),
+        Genre("сёдзё-ай", "el_1320"),
+        Genre("сёнэн", "el_1326"),
+        Genre("сёнэн-ай", "el_1330"),
+        Genre("спорт", "el_1321"),
+        Genre("сэйнэн", "el_1329"),
+        Genre("сянься", "el_6631"),
+        Genre("трагедия", "el_1344"),
+        Genre("триллер", "el_1341"),
+        Genre("ужасы", "el_1317"),
+        Genre("уся", "el_6632"),
+        Genre("фэнтези", "el_1323"),
+        Genre("школа", "el_1319"),
+        Genre("эротика", "el_1340"),
+        Genre("этти", "el_1354"),
+        Genre("юри", "el_1315"),
+        Genre("яой", "el_1336"),
+    )
+
+    override fun setupPreferenceScreen(screen: androidx.preference.PreferenceScreen) {
+        super.setupPreferenceScreen(screen)
+        EditTextPreference(screen.context).apply {
+            key = DOMAIN_TITLE
+            this.title = DOMAIN_TITLE
+            summary = domain
+            this.setDefaultValue(DOMAIN_DEFAULT)
+            dialogTitle = DOMAIN_TITLE
+            setOnPreferenceChangeListener { _, newValue ->
+                try {
+                    val res = preferences.edit().putString(DOMAIN_TITLE, newValue as String).memit()
+                    Toast.makeText(screen.context, "Для смены домена необходимо перезапустить приложение с полной остановкой.", Toast.LENGTH_LONG).show()
+                    res
+                } catch (e: Exception) {
+                    e.printStackTrace()
+                    false
+                }
```

Comment:
```suggestion
                Toast.makeText(screen.context, "Для смены домена необходимо перезапустить приложение с полной остановкой.", Toast.LENGTH_LONG).show()
                true
```
build failed here btw, also you don't need to manually set the preference value, app will take care of it.

Reply:
What the heck? The reason of the error is simple: it changed "commit" function name to "memit" for some reason. The latter function doesn't exist. What's even stranger is that everything's fine in my edited code, but not in the commit.